lib/runtime/test_helpers.go

```diff
@@ -80,30 +80,24 @@ func GetAbsolutePath(targetDir string) string {
 }
 
 // GetRuntimeBlob checks if the test wasm @testRuntimeFilePath exists and if not, it fetches it from @testRuntimeURL
-func GetRuntimeBlob(testRuntimeFilePath, testRuntimeURL string) (n int64, err error) {
+func GetRuntimeBlob(testRuntimeFilePath, testRuntimeURL string) error {
 	if utils.PathExists(testRuntimeFilePath) {
-		return 0, nil
+		return nil
 	}
 
-	out, err := os.Create(testRuntimeFilePath)
+	/* #nosec */
```

Comment:
What does this do? 🤔

Reply:
This comment is there before I just keep it haha and I think is just a comment

Reply:
Ah ok ! You can try removing it. That's probably a comment to disable linting of a previous linter we used to use.

Reply:
Removed!